components/application-configuration/api/src/main/java/org/phenotips/configuration/internal/consent/ConsentAuthorizer.java

```diff
@@ -54,8 +54,7 @@
     List<RecordElement> filterForm(List<RecordElement> elements, Patient patient);
 
     /**
-     * Determines if any interactions with a patient record given the consents that have been granted should be
-     * allowed.
+     * Determines if all required (by the system) consents have been granted for the given patient record.
      *
      * @param patient record in question
      * @return {@link true} if the patient record can be modified or viewed, {@link false} otherwise
```

Comment:
`{@link true}` is invalid, it breaks the javadoc generation. This should be `{@code true}`. Same for `false`, and for all the other places where this is used.